tests/upgrade/test_crossgrade.sh

```diff
@@ -43,8 +43,13 @@ usage() {
 }
 
 ISTIO_NAMESPACE="istio-system"
-# Minimum % of all requests that must be 200 for test to pass.
-MIN_200_PCT_FOR_PASS="85"
+# Maximum % of 503 response that cannot exceed
+MAX_503_PCT_FOR_PASS="15"
+# Maximum % of connection refused that cannot exceed
+# Set it to high value so it fails for explicit sidecar issues
+MAX_CONNECTION_ERR_FOR_PASS="30"
+SERVICE_UNAVAILABLE_CODE="Code 503"
```

Comment:
The name might be a bit misleading `SERVICE_UNAVAILABLE_CODE` suggests to me that there is only a number (503) assigned to it. How about `SERVICE_UNAVAILABLE_KEYWORD`?

Reply:
ok it makes sense, updated to *_KEYWORD